### PR TITLE
Feat aspect ratio

### PR DESCRIPTION
**Pull request by manuelroth**
- This PR implements the height calculation based on displayOption `size`
- This branch is deployed on staging, so it can be tested in Staging LivingDocs -> https://edit-stage.nzz.ch/articles/15611/edit/add
  - The `size` option is only available for charttypes `line`, `column` and `stacked-column`
  - In a regular article (width of 560px) charts with `size` `basic` have a height of `240px` and with axis label `260px`
  - With `size` `prominent` they have a height of `315px` and with axis label `335px`
- Closes https://github.com/nzzdev/Q-chart/issues/99

### CODE REVIEW

File: routes/rendering-info/web-svg.js [hunk 1]
```diff
@@ -52,28 +51,31 @@ function getSpecConfig(item, baseConfig, toolRuntimeConfig) {
...
-
   const chartTypeConfig = require(`../../chartTypes/${chartType}/config.js`);
   if (chartTypeConfig.data.handleDateSeries) {
     // if we have a date series, we change the date values to date objects
-    // and set the detected dateFormat to the mappingConfig to be used within the mapping functions
+    // and set the detected dateFormat to the renderingInfoInput to be used within the mapping functions
     if (dateSeries.isDateSeriesData(item.data)) {
-      mappingConfig.dateFormat = dateSeries.getDateFormatForData(item.data);
-      item.data = dateSeries.getDataWithDateParsed(item.data);
+      renderingInfoInput.dateFormat = dateSeries.getDateFormatForData(
```
Comment (benib) [inline routes/rendering-info/web-svg.js]: Mmh, somehow the wording `renderingInfoInput` doesn't really match for this. Not sure what to do about it, maybe we find a better wording for this.
Somehow I read `input` as something that is coming from the outside into the renderingInfo route. This works great for the item and toolRuntimeConfig. Not so much for things that get added inside the tool.

File: resources/display-options-schema.json [hunk 2]
```diff
@@ -6,6 +6,22 @@
     "hideTitle": {
       "type": "boolean",
       "title": "Titel ausblenden"
+    },
+    "size": {
+      "title": "Gewicht",
+      "type": "string",
+      "default": "basic",
+      "enum": ["basic", "prominent"],
+      "Q:options": {
```
[the remark this answers is not in the document]
Comment (manuelroth) [reply resources/display-options-schema.json]: Changed in https://github.com/nzzdev/Q-chart/pull/101/commits/906deaed8bdaf9ad1f4477d0197423e85143b6dd

File: routes/rendering-info/web-svg.js (hunk 1 above)
Comment (manuelroth) [reply routes/rendering-info/web-svg.js]: Yes its true. Maybe `mappingData` could be a better name.

Comment (manuelroth) [reply routes/rendering-info/web-svg.js]: I renamed the property `renderingInfoInput` to `mappingData` -> https://github.com/nzzdev/Q-chart/pull/101/commits/525b356fcc43d9059172c1edea4c579ed8b8ba27

**Pull request merged**